src/care/host_ptr.h

```diff
@@ -77,7 +77,7 @@ namespace care {
          ///
          /// Construct from chai::ManagedArray
          ///
-         host_ptr(chai::ManagedArray<T> const &ptr) : m_ptr(ptr.getPointer(chai::CPU)) {}
+         host_ptr(chai::ManagedArray<T> ptr) : m_ptr(ptr.getPointer(chai::CPU)) {}
```

Comment:
Likewise, add a TODO comment saying that this should be reverted to const in a future update of chai

Reply:
Done.